src/chaplin/composer.coffee

```diff
@@ -3,9 +3,9 @@
 _ = require 'underscore'
 Backbone = require 'backbone'
```

Comment:
I've just stumble upon the `external` option of rollup, I'll change this one in a future commit